common/util.go

```diff
@@ -128,9 +130,39 @@ func BytesToVideoProfile(txData []byte) ([]ffmpeg.VideoProfile, error) {
 	return profiles, nil
 }
 
+func DefaultProfileName(width int, height int, bitrate int) string {
+	return fmt.Sprintf("%dx%d_%d", width, height, bitrate)
+}
+
+func FFmpegProfiletoNetProfile(ffmpegProfiles []ffmpeg.VideoProfile) ([]*net.VideoProfile, error) {
+	profiles := []*net.VideoProfile{}
+	for _, profile := range ffmpegProfiles {
+		width, height, err := ffmpeg.VideoProfileResolution(profile)
+		if err != nil {
+			return nil, err
+		}
+		br := strings.Replace(profile.Bitrate, "k", "", 1)
+		bitrate, err := strconv.Atoi(br)
+		if err != nil {
+			return nil, err
+		}
+		name := profile.Name
+		if name == "" {
+			profile.Name = "ffmpeg_" + DefaultProfileName(width, height, bitrate)
+		}
+		fullProfile := net.VideoProfile{
+			Name:    profile.Name,
+			Width:   int32(width),
+			Height:  int32(height),
+			Bitrate: int32(bitrate),
+			Fps:     int32(profile.Framerate),
+		}
+		profiles = append(profiles, &fullProfile)
+	}
+	return profiles, nil
+}
+
 func ProfilesToTranscodeOpts(profiles []ffmpeg.VideoProfile) []byte {
-	//Sort profiles first
-	sort.Sort(ffmpeg.ByName(profiles))
```

Comment:
@yondonfu Note this change - it was leading to some unexpected side effects during our testing. This change was a quick workaround for that, but it does affect how sigs are generated. (See this commit: https://github.com/livepeer/go-livepeer/pull/1242/commits/6ebfbe54363020b1414c9f4bf1e1cd87a3dc5431) Figured it'd be OK since we're reworking the sigs with https://github.com/livepeer/go-livepeer/issues/1091 anyway, but wanted to check in with you first.

Reply:
Thanks for the heads up - think it should be ok since we'll be reworking things in #1091